README.md

```diff
@@ -17,309 +17,125 @@ Supports [CANNON.js](http://schteppe.github.io/cannon.js/) and [Ammo.js](https:/
+
+Specific details  vary between drivers, so you should consult driver-specific documentation for details.  Since Cannon.js is written in native Javascript, its API is generally easier to use, and problems are simpler to debug.  In comparison, making use of the Ammo.js APIs can be quite hard work (there's definitely scope to improve the available documentation and examples here!)  
+
+- [Cannon.js](CannonDriver.md)
+
+- [Ammo.js](AmmoDriver.md)
+
+  
+
+## Driver-specific limitations
 
-```html
-<!-- The debug:true option creates a wireframe around each physics body. If you don't see a wireframe,
-     the physics system may be unable to parse your model without a shape:box or shape:hull option. -->
-<a-scene physics="debug: true">
+This is a list of limitations that has been oberved with particular drivers (and also with physx).  It's intended to provide a checklist to help developers to choose between physics drivers for a particular project, so they don't pick a driver that turns out to be missing some feature that is fundamental for their application.
 
-  <!-- Camera -->
-  <a-entity camera look-controls></a-entity>
+This list is probably incomplete, so if you find an additional significant limitation, please add it to this list.
 
-  <!-- Floor -->
-  <a-plane static-body></a-plane>
+### Cannon.js
 
-  <!-- Immovable box -->
-  <a-box static-body position="0 0.5 -5" width="3" height="1" depth="1"></a-box>
+- Can't handle collision with fast moving bodies, as it does not offer Continuous Collision Detection (CCD)
+- No support for collision filtering - all object pairs 
+- Restitution (bounciness) is a global property, rather than per-body
+- No stateful collision data - just a stream of events that starts when collision is happening, and stops when collision ends (compare Ammo, which offers distinct collide-start, collide-end events and collision state that can be queried at any time; no idea yet what PhysX offers here...)
 
-  <!-- Dynamic box -->
-  <a-box dynamic-body position="5 0.5 0" width="1" height="1" depth="1"></a-box>
-
-</a-scene>
-```
 
-## Components
 
-### `dynamic-body` and `static-body`
+### Ammo.js
 
-The `dynamic-body` and `static-body` components may be added to any `<a-entity/>` that contains a mesh. Generally, each scene will have at least one `static-body` for the ground, and one or more `dynamic-body` instances that the player can interact with.
+- No support for off-center attachment of constraints to bodies (integration issue?)
 
-- **dynamic-body**: A freely-moving object. Dynamic bodies have mass, collide with other objects, bounce or slow during collisions, and fall if gravity is enabled.
-- **static-body**: A fixed-position or animated object. Other objects may collide with static bodies, but static bodies themselves are unaffected by gravity and collisions.
+- No support for slider constraint (integration issue?)
```

Comment:
maybe a quick comment explaining what a slider is

Reply:
done.  Like a bead on an abacus.

Reply:
Not sure if there was a mistake, but I don't see this update.

Reply:
Maybe I forgot to push.

https://github.com/c-frame/aframe-physics-system/pull/11/commits/c712774436ed3deeadd3d634e3c1b36a421b0bf0